lib/graphql/account.ts

```diff
@@ -28,4 +49,14 @@ export class Account extends Model<AccountModel> {
     const result: Query = await this.client.rawQuery(GET_ACCOUNT);
     return result.viewer?.mainAccount || null;
   }
+
+  /**
+   * Different statistics to the users account balance e.g. yours, unknow, vatTotal, taxMissing ...
+   *
+   * @returns current user account statistics
+   */
+  public async getAccountStats(): Promise<AccountStats | null> {
```

Comment:
was not sure how to name it.
now its used like `account.getAccountStats()` maybe just `account.getStats()` is better? 🤔

Reply:
I think `getStats` would be better for our SDK 🙂 We usually don't repeat model name in methods, e.g. `transaction.categorize`, `transfer.createOne`... there are exceptions but let's try to keep consistency higher by using `getStats` name 👍